gravityflow/gravityflowwoocommerce
# Fix PHP fatal error is thrown when creating entries in forms with the Payment Step

This PR fixes an issue where PHP fatal is thrown when creating entries in forms with the Payment Step.

It was introduced by PR #10, where we add a second parameter `$order_id` to the `can_create_entry_for_order ()` method which defaults to `null`. The PHP fatal was created in the Payment Step because it tried to get the `$order_id` but the `$order` object isn't available in this test case (we generate the entry directly, but not create entries with WooCommerce orders).

## Screenshots
![image](https://user-images.githubusercontent.com/12166/68073225-b4cb0000-fdc8-11e9-900a-a7597c98ebdd.png)

## Testing instructions
1. Using this [sample form](https://github.com/gravityflow/gravityflowwoocommerce/files/3800638/WC-pay-later.json.zip) which contains a Payment Step.
2. Submit the form to create new entries. Go to Workflow inbox to view entry details, and approve the entry.
3. You'll see the PHP fatal error right after the entry is approved. Switch to this branch should fix it.
